### PR TITLE
Add SVG support

### PR DESCRIPTION
**Pull request by flosse**

### CODE REVIEW

**Comment by MartinKavik** [comment]
Thanks, I'll try to look at it on Monday.

**Comment by MartinKavik** [comment]
I don't know how to run the example. I see extra `index.html` and missing `Makefile.toml`. We should be able to run all examples the same way, described in [development.md](https://github.com/MoonZoon/MoonZoon/blob/main/docs/development.md#3-start-example)

**Comment by flosse** [comment]
> I don't know how to run the example.

I just ran `../../target/debug/mzoon start`

> I see extra `index.html` and missing `Makefile.toml`. We should be able to run all examples the same way, described in [development.md](https://github.com/MoonZoon/MoonZoon/blob/main/docs/development.md#3-start-example)

I'm sorry, didn't read that :see_no_evil: ... it should work now

File: crates/mzoon/src/main.rs [hunk 1]
```diff
@@ -320,6 +320,7 @@ fn start_backend_watcher(
 }
 
 fn generate_backend_build_id() {
+    fs::create_dir_all("backend/private").unwrap();
```
Comment (MartinKavik) [inline crates/mzoon/src/main.rs:323]: Please remove this line and add a `private` directory with `.gitignore` to the `svg` example. Build id, certificates and other stuff is stored in `private` and you don't want to accidentally push it to Git (for security reasons, unnecessary commit changes, incompatible certs among envs, etc.)

[private](https://github.com/MoonZoon/MoonZoon/tree/main/examples/counter/backend/private) in `counter` example

**Review by MartinKavik — CHANGES_REQUESTED** [review]
One review comment added. Once resolved I think we can merge it :)

**Comment by MartinKavik** [comment]
Ouch, sorry I forgot you've formatted the code and merged a new [PR](https://github.com/MoonZoon/MoonZoon/pull/10) 🙈 I hope rebasing won't be too painful 🤯

**Review by MartinKavik — APPROVED** [review]

**Pull request merged**